src/actions/authActions.js

```diff
@@ -0,0 +1,15 @@
+import { GET_TOKEN , SET_TOKEN} from "../type";
+
+export const getToken = (code) => {
+  return {
+    type: GET_TOKEN,
+    payload: code,
+  };
+};
```

Comment:
```suggestion
export const getToken = payload => {
    type: GET_TOKEN,
    payload
};
```
Try to use shorthand syntax whenever possible. Update everywhere applicable